docs/components/NavigationDrawers.md

```diff
@@ -61,3 +67,48 @@ OdsModalDrawer(
     // Put here the rest of the UI content
 }
 ```
+
+#### OdsModalDrawer API
+
+Parameter | Default&nbsp;value | Description
+-- | -- | --
+<b>`header: `</b>[OdsModalDrawerHeader](#odsmodaldrawerheader) | | Content descriptor of the drawer header
+<b>`drawerItems: List<`</b>[OdsModalDrawerItem](#odsmodaldraweritem)<b>`>`</b> | | List of `OdsModalDrawerItem` displayed in a column inside the modal drawer
+`modifier: Modifier` | `Modifier` | `Modifier` applied to the modal drawer
+`state: DrawerState` | `rememberDrawerState(DrawerValue.Closed)` | State of the modal drawer
+`selectedItem: OdsModalDrawerListItem?` | `null` | Selected `OdsModalDrawerListItem` that appears in selected state
+`onItemClick: (OdsModalDrawerListItem) -> Unit` | `{}` | Callback invoked on an `OdsModalDrawerListItem` click. Provides the clicked `OdsModalDrawerListItem`.
+`content: @Composable () -> Unit` | | Content of the rest of the UI
+{:.table}
+
+#### API descriptors
```

Comment:
I am not sure descriptor is the proper name for these classes but on the other hand I don't know how to call them.